package.json

```diff
@@ -34,7 +35,7 @@
     "highlight.js": "^9.5.0",
     "http-proxy": "^1.14.0",
     "ip": "^1.1.3",
-    "knex": "*",
+    "knex": "^0.16.3",
```

Comment:
This should be left to * to get always the latest version for bundling knex and getting the changelog...

Reply:
Don't remember why I changed it, maybe it was accidental. Changed back.

Reply:
Maybe some VSCode or npm stuff does that automatically... I remember having to fix that  every now and then when I've been updating docs.

Reply:
Could be :)